CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 13.1.0
+
+- Changed: Add `allow` option to `no-console` rule.
```

Comment:
```suggestion
- Added: `allow` option to `no-console` rule.
```

The "Changed" descriptor, along with the "Removed" one, is [usually associated with a major release](https://keepachangelog.com/en/1.0.0/#how).

Reply:
Oh, it's difficult... 😅 Thank you!